core/schema/src/main/java/org/opennms/core/schema/Migrator.java

```diff
@@ -452,14 +452,20 @@ public void createUser() throws MigrationException {
         try {
             c = m_adminDataSource.getConnection();
             st = c.createStatement();
-            st.execute("CREATE USER " + getDatabaseUser() + " WITH PASSWORD '" + getDatabasePassword() + "'");
+            st.execute("CREATE USER " + getUserForDDL() + " WITH PASSWORD '" + getDatabasePassword() + "'");
         } catch (final SQLException e) {
             throw new MigrationException("an error occurred creating the OpenNMS user", e);
         } finally {
             cleanUpDatabase(c, null, st, rs);
         }
     }
 
+    protected String getUserForDDL() {
```

Comment:
DDL as in Data Definition Language (DDL)?

Reply:
Yes